yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -137,12 +142,12 @@ def decrypt(data):
         }, query=query, fatal=fatal)
         return api_json, decrypt
 
-    def _download_metadata(self, url, video_id, lang, props_key):
-        metadata = self._call_api(video_id, msg='metadata', query={'al': lang or 'ja'}, auth=False, fatal=False)
+    def _download_metadata(self, url, video_id, lang, props_keys):
+        metadata = self._call_api(video_id, msg='video information', query={'al': lang or 'ja'}, auth=False, fatal=False)
```

Comment:
revert. this would change the message for wrestleuniverse as well, and 'metadata' is just as accurate, no?
```suggestion
        metadata = self._call_api(video_id, msg='metadata', query={'al': lang or 'ja'}, auth=False, fatal=False)
```